src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,240 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+  Select,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { duplicateInstance, fetchInstance, startInstance } from "api/instances";
+import { isClusteredServer } from "util/settings";
+import { useSettings } from "context/useSettings";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchProjects } from "api/projects";
+import { fetchStoragePools } from "api/storage-pools";
+import { fetchClusterMembers } from "api/cluster";
+import { Link } from "react-router-dom";
+
+interface Props {
+  instance: LxdInstance;
+  close: () => void;
+}
+
+export interface DuplicateInstanceFormValues {
+  instanceName: string;
+  project: string;
+  targetClusterMember?: string;
+  storagePool?: string;
+  allowInconsistent?: boolean;
+  instanceOnly?: boolean;
+  stateful?: boolean;
+}
+
+const DuplicateInstanceForm: FC<Props> = ({ instance, close }) => {
+  const eventQueue = useEventQueue();
+  const toastNotify = useToastNotification();
+  const { data: settings } = useSettings();
+  const isClustered = isClusteredServer(settings);
+
+  const { data: projects = [] } = useQuery({
+    queryKey: [queryKeys.projects],
+    queryFn: fetchProjects,
+  });
+
+  const { data: storagePools = [] } = useQuery({
+    queryKey: [queryKeys.storage],
+    queryFn: () => fetchStoragePools(instance.project),
+  });
+
+  const { data: clusterMembers = [] } = useQuery({
+    queryKey: [queryKeys.cluster],
+    queryFn: fetchClusterMembers,
+  });
+
+  const notifySuccess = (instanceName: string) => {
+    const newInstance = (
+      <Link to={`/ui/project/${instance.project}/${instanceName}`}>
+        {instanceName}
+      </Link>
+    );
+    toastNotify.success(
+      <>
+        Created and started cloned instance <b>{newInstance}</b>.
+      </>,
+    );
+  };
+
+  const formik = useFormik<DuplicateInstanceFormValues>({
+    initialValues: {
+      instanceName: instance.name + "-1",
+      project: "default",
+      allowInconsistent: false,
+      instanceOnly: false,
+      stateful: false,
+    },
+    validationSchema: Yup.object().shape({
+      instanceName: Yup.string().required(),
```

Comment:
we should validate that instance name does not exist for the selected target project. See `InstanceSchema` in `CreateInstance.tsx` for reference.